service/history/decisionTaskHandler.go

```diff
@@ -837,6 +840,70 @@ func (handler *decisionTaskHandlerImpl) handleDecisionSignalExternalWorkflow(
 	return nil
 }
 
+func (handler *decisionTaskHandlerImpl) handleDecisionUpsertWorkflowSearchAttributes(
```

Comment:
plz follow the indentation rule, like this
```
func (handler *decisionTaskHandlerImpl) handleDecisionSignalExternalWorkflow(
	attr *workflow.SignalExternalWorkflowExecutionDecisionAttributes,
) error {
```

Reply:
ok